po/ru.po

```diff
@@ -1065,40 +1075,59 @@ msgid ""
 "  mutexes) and async/await concurrency (cooperative multitasking using\n"
 "  futures)."
 msgstr ""
+"* [Android](android.md): рассчитанный на полдня курс по использованию Rust для разработки на\n"
+"платформе Android (AOSP).\n"
+"Сюда входит взаимодействие с C, C++ и Java.\n"
+"* [Bare-metal](bare-metal.md): однодневное занятие по использованию Rust для низкоуровневой\n"
+"(embedded) разработки, охватывающее как микроконтроллеры, так и обычные процессоры.\n"
+"* [Concurrency](concurrency.md): полный день занятий по изучению конкуретности в Rust.\n"
+"Мы рассмотрим как классическую конкурентность\n"
+"(вытесняющая многозадачность с использованием потоков и мьютексов),\n"
+"так и async/await конкурентность (кооперативная многозадачность с использованием футур)."
```

Comment:
Мы рассмотрим как классическую вытесняющую многозадачность с использованием потоков и взаимного исключения, так и кооперативную многозадачность async/await с использованием фьючерсов\футур (Future)

Reply:
This is another tricky case. IMO `concurrency !=  многозадачность`. 
I can see how `конкурентность` sounds a bit weird, but it's not something I just invented. I see people in the Rust community use this term. I would prefer to go with current definition

Reply:
Oxford dictionary:
Concurrency: the ability to execute more than one program or task simultaneously.

How is this not a многозадачность?

Reply:
Yeah, the problem is that with this definition from the Oxford dictionary the term `параллелизм` fits just fine

Reply:
Nope. Parallelism is dividing ONE TASK into pieces and execution of those pieces OF THE SAME TASK simultaneously in parallel.

Reply:
Right, but the Cambridge dictionary thinks that it's a `the ability of a computer to do two or more pieces of work at the same time`. Pretty similar to the definition of the `concurrency` you've provided before. 

We are entering into a controversy that I would very much like to avoid

Reply:
I am teaching concurrency in a University. You can trust me.